popSimManu.tex

```diff
@@ -122,26 +122,26 @@
 independently, leading to duplication of effort and the possibility for error.
 Population genetics, as a field, also lacks standard benchmarks by which new tools for inference might
 be measured. Here we describe a new resource, \stdpopsim, that attempts to rectify this situation.
-\Stdpopsim is a community-driven open source project, which provides a standard
+\Stdpopsim is a community-driven open source project, which provides easy access to a standard
 catalog of published simulation models from a wide range of organisms
 and supports multiple simulation engine backends.
 % PopSim assures correctness in our implementations through a mature, community-based quality control pipeline.
 We share some examples demonstrating how \stdpopsim
 can be used to systematically compare demographic inference methods,
 and we encourage an even broader
-community of developers to join in the effort to standardize
-population genetic methods.}
+community of developers to contribute to this growing resource.
+}
 \date{}
 
-\keywords{Population genetics, Simulation, Inference}
+\keywords{Population genetics, Simulation, Inference, Reproducibility}
```

Comment:
should we add "Standardization" or "Standards"?

Reply:
is that a keyword? dunno.